src/test/java/arb/testutil/ProjectBuilder.java

```diff
@@ -0,0 +1,62 @@
+package arb.testutil;
+
+import arb.model.project.Deadline;
+import arb.model.project.Project;
+import arb.model.project.Title;
+
+/**
+ * A utility class to help with building Project objects.
+ */
+public class ProjectBuilder {
+
+    public static final String DEFAULT_TITLE = "Large Tree";
+    public static final String DEFAULT_DEADLINE = "2000-01-01";
+
+    private Title title;
+    private Deadline deadline;
+
+    /**
+     * Creates a {@code ProjectBuilder} with the default details.
+     */
+    public ProjectBuilder() {
+        title = new Title(DEFAULT_TITLE);
+        deadline = new Deadline(DEFAULT_DEADLINE);
+    }
+
+    /**
+     * Initializes the ProjectBuilder with the data of {@code personToCopy}.
```

Comment:
Maybe you can change `personToCopy` to `projectToCopy` to fit the method parameter?